src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdminService.java

```diff
@@ -318,6 +322,12 @@ private String formatNotificationTitle(TseAdminRecordDecisionType decision) {
                 String.format(DECISION_NOTIFICATION_TITLE, decision.getEnterNotificationTitle());
     }
 
+    private String formatDecisionDetails(TseAdminRecordDecisionType decision) {
```

Comment:
Consider putting these kind of methods into a helper class rather than having to suppress warnings

Reply:
Thank you for comment. Will fix this in another PR.

Reply:
I have raised a Tech Deby (RET-3370) to resolve PMD.TooManyMethods